### PR TITLE
Swapneel refactor

### PR DESCRIPTION
**Pull request by swapneelap**
My nitpicky refactor: no need to accept all the changes!

### CODE REVIEW

File: src/mochada_kit/__init__.py [hunk 1]
```diff
@@ -9,139 +9,10 @@
...
-    if args.config:
-        make_config(puml_path=args.puml_path)
-
-
-load_config()
-
-if __name__ == "__main__":
-    main()
+_THEMES_DIR = (
+    pathlib.Path(__file__).absolute().parent.parent.joinpath("themes").as_posix()
```
Comment (tgwoodcock) [inline src/mochada_kit/__init__.py]: Here we need to go up one more parent to get to the themes dir.

File: src/mochada_kit/cli.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+#         type=str,
+#         required=False,
+#         const=None,
+#         help="Full path to plantuml.jar (default: %(default)s)",
+#     )
+#
+#     args = parser.parse_args()
+#
+#     if args.config:
+#         make_config(puml_path=args.puml_path)
```
Comment (tgwoodcock) [inline src/mochada_kit/cli.py]: Let's remove all this old stuff,

File: src/mochada_kit/config.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+def read_config():
+    if config_file_path := pathlib.Path.home().joinpath(".mochada_kit", "config.json"):
+        with open(config_file_path) as handle:
+            return json.load(handle)
+    else:
+        error = (
+            f"Cannot find configuration file at: {config_file_path}."
+            "To fix, please run: mochadakit -c -p <path to plantuml.jar>"
+        )
+        raise OSError(error)
```
Comment (tgwoodcock) [inline src/mochada_kit/config.py]: How about downgrading the error to a warning and returning None? That way, we avoid the case that the user is forced to set the config in order to be able to import the module mochada_kit.running.

We should also update the error/warning message to the new format of your cli (the flag -c is no longer valid and we need the argument config instead). I would also like to have the name of the cli command as "mochada_kit" (with underscore, just to keep it the same as the package name).

File: pyproject.toml [hunk 4]
```diff
@@ -83,10 +83,7 @@ known-local-folder = ["src"]
 'content-type' = "text/markdown"
 
 [project.scripts]
-mochada_kit = "mochada_kit.__init__:main"
-
-[tool.setuptools.dynamic]
-version = { attr = "mochada_kit.__version__" }
+mochadakit = "mochada_kit.cli:cli"
```
Comment (tgwoodcock) [inline pyproject.toml]: I'd like to have the name of the project script as "mochada_kit" with the hyphen, just to keep it the same as the package name.

File: examples/make_workflow_diagram.ipynb [hunk 5]
```diff
@@ -96,14 +96,14 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "D:/Work/Projects/MaMMoS/CHADA_MODA/mochada_kit/themes\n"
+      "/home/swapneel/Projects/Forks/mochada_kit/src/themes\n"
```
Comment (tgwoodcock) [inline examples/make_workflow_diagram.ipynb:99]: I don't know if this matters here but the themes dir is actually outside the src dir (see other comment about the extra parent for the _THEMES_DIR in __init_.py).

**Review by tgwoodcock — CHANGES_REQUESTED** [review]
Fantastic stuff @swapneelap! Love the changes - they make for a much cleaner interface.

I just have a few small comments which need minor changes and we can discuss about these.

**Comment by swapneelap** [comment]
Hello @tgwoodcock, I have ad the changes in according to your comments. If you think they are good, please merge the pull request.

**Pull request merged**